lldap/lldap
# github: Leverage metadata

* Reduce mouthfull of image tagging.
* Tags
  * Debian
  ```
  vX-debian
  vX.Y-debian
  vX.Y.Z-debian
  stable-debian
  latest-debian
  ```
  * Alpine
  ```
  vX-alpine
  vX.Y-alpine
  vX.Y.Z-alpine
  stable-alpine
  latest-alpine
  latest
  stable
  ```
* Ensure database startup correctly on init test job
* Ensure database startup correctly on migration test job
